docker/vlang/Dockerfile.alpine

```diff
@@ -2,25 +2,29 @@
 # - No UI dependencies
 # - gcc compiler (only works on alpine)
 
+ARG USER=thevlang
+
 # Stage 1 - Build the V-compiler
-FROM thevlang/vlang:alpine-build AS builder
+FROM ${USER}/vlang:alpine-build AS builder
```

Comment:
this allows to test when you do a fork easily.

Reply:
Yes but I use my own username in secrets so it will not tag correctly